packages/cli/src/cmds/preview/preview.ts

```diff
@@ -532,7 +602,7 @@ export default class Preview extends YargsCommand {
 
   private async startNgrok(
     workspaceFolder: string,
-    depsChecker: CliDepsChecker
```

Comment:
Any code clean up in the old deps checker?

Reply:
Removed and add UT